about.html

```diff
@@ -0,0 +1,135 @@
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"/>
+  <link rel="stylesheet" href="styles.css" type="text/css">
+  <link href="https://fonts.cdnfonts.com/css/tasty-food" rel="stylesheet">
+</head>
+<body>
+    <section class="intro-container"> 
+        <header>
+            <nav>
+              <div class="mobile hamburger-icon">
+                <div class="drop-menu hidden-menu"> 
+                    <div class="close-button">X</div>
+                    <ul class="nav-menu">
+                        <li class="nav-menu-item"><a href="./index.html" class="nav-link">Home</a></li>
+                        <li class="nav-menu-item"><a href="#" class="nav-link">Program</a></li>
+                        <li class="nav-menu-item"><a href="#" class="nav-link">Presenters</a></li>
+                        <li class="nav-menu-item"><a href="#" class="nav-link">Partners</a></li>
+                    </ul>    
+                </div>
+                <i class="fas fa-bars"></i>
+              </div>
+            </nav>
+            <div class="mobile nav-line"></div>
+                    <div class="top-bar">
+            <ul>
+                <li><a href="#"><img class="social-icon" src="./images/facebook.png"></a></li>
+                <li><a href="https://www.facebook.com/mchafu"><img class="social-icon" src="./images/twitter.png"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="https://thetjaypod.wordpress.com/">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+
+            </ul>
+        </div>
+            <div class="desk-nav">
+                <a class= "imagea" href="./index.html"><img class="logo" src="./images/Shakys-logo.jpeg"></a>
+                <ul class="desk-nav-menu">
+                    <li class="desk-nav-menu-item"><a class="desk-nav-link" href="./index.html">Home</a></li>
+                    <li class="desk-nav-menu-item"><a class="desk-nav-link" href="#">Program</a></li>
+                    <li class="desk-nav-menu-item"><a class="desk-nav-link" href="#">Speakers</a></li>
+                    <li class="desk-nav-menu-item"><a class="desk-nav-link" href="#">Partners</a></li>
+                </ul>
+                <div class="register-now">Register Now</div>
+            </div>
+    
+          </header>
+    <div class="about-page-info">
+        <div class="headings">
+            <h2 class="quote">"Good Food Unites Hearts, Nourishes Souls."</h2>
+            <h1 class="topic">SHAKY'S ANNUAL COOKOUT 2023</h1> 
+            </div>
+        
+        <div class="home-message">
```

Comment:
- [ ] please kindly centralize the page title, a text block with the page description in the about page
- [ ] please remove the small heading(Good Food Unites Hearts, Nourishes Souls) as the about page has just the main title
- [ ] please ensure the email is on its own and underlined like in the original design
- [ ] kindly add the necessary gaps to make the designs look alike
yours

![Screenshot (815)_LI](https://user-images.githubusercontent.com/65298048/219622514-cbd5db80-25a9-400f-aacf-392ab758e632.jpg)
original
![Screenshot (814)](https://user-images.githubusercontent.com/65298048/219623158-939e0f24-283d-4cbd-86be-b57058f4a618.png)